### DISCUSSION TITLE
status 200 on redirect

### DISCUSSION THREAD

**Discussion by ndrswlkr** [mojolicious/mojo.js#116] [Q&A] [answered]
i redirect to the loggin page when the user is logged out.
i do this with await ctx.redirect('/login')
in the frontend i try to handle this by checking if  res.status is >= 300 && res.status <= 400
but the status is allways 200 even if i ctx.redirect('/login', {status: 301})
what am i doing wrong?

**Comment by kraih** [comment]
Browsers hide 3xx responses usually, and you will only see the final response.

**Comment by ndrswlkr** [comment]
This Leads to a follow up question.
I serve my app from uberspace.de, where i listen to http://0.0.0.0:port from mojojs, but the app is accessed through https. The redirects jedoch, arive as http and get rejected by the Browser.
Is there a way to solve this?

**Reply by kraih** [reply, marked as answer]
There is context information missing here. But it suspect that you need to configure the app to work with a reverse proxy server. https://mojojs.org/docs/Cookbook.md#reverse-proxy

**Reply by ndrswlkr** [reply]
Yes that was it. --proxy did the Trick
Thank jou!
